.github/workflows/tests.yml

```diff
@@ -71,21 +47,7 @@ jobs:
         # Install distributed to run importance_tests/test_init tests.
         pip install distributed
 
-    - name: Output installed packages
-      run: |
-        pip freeze --all
-
-    - name: Output dependency tree
-      run: |
-        pip install pipdeptree
-        pipdeptree
-
     - name: Tests
-      run: |
-        pytest tests \
-          --ignore tests/test_mxnet.py
-
-    - name: Tests(Deprecated)
       run: |
         pip install "numpy<1.24"
-        pytest tests/test_mxnet.py
```

Comment:
Is `test_mxnet.py` no longer used?
Then, can it be removed from the repository?

Reply:
This PR represents the initial phase of dividing integration tests. The tests will be separated into individual integrations. For simplicity, I have removed the process specific to a particular integration.
I believe this process should be implemented in the MXNet tests (e.g., `test_mxnet.yml`)